src/components/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {ChangeEvent, ReactNode} from "react";
+import {makeStyles, withStyles} from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import {SelectProps} from "@material-ui/core/Select";
+import Select from '@material-ui/core/Select';
+import InputBase from '@material-ui/core/InputBase';
+import ArrowDropDownRoundedIcon from '@material-ui/icons/ArrowDropDownRounded';
+import {BootStrapInputStyle, DropdownStyle, SelectStyle} from "./SelectInput.styles";
+
+const BootstrapInput = withStyles(BootStrapInputStyle)(InputBase);
+
+const useStyles = makeStyles(DropdownStyle)
+
+
+const SelectStyled = withStyles(SelectStyle)(Select);
+
+export interface option {
+  label: string;
+  value: any;
+}
+
+export interface SelectInputProps extends SelectProps {
+  options: option[]
+}
+
+export default function SelectInput({options, ...props}: SelectInputProps) {
+  const [value, setValue] = React.useState<any>('');
```

Comment:
I think we should have `value` and `onChange` be left to props. That way the code that implements this can control what the value is and what happens when a value is changed (potentially run a side effect like an API call, etc)

Reply:
hmm yeah good idea, will change